hrjobcontract/js/src/job-contract/services/contract-health.js

```diff
@@ -60,7 +61,22 @@ define([
 
             deffered.resolve(data || {});
           } else {
-            //TODO call2API
+            params.sequential = 1;
+            params.field = "hrjobcontract_health_life_insurance_plan_type";
```

Comment:
The method is called `getOptions`, but you're limiting the query to just this field. If i was a consumer of this method i would be pretty confused as to why something as generic as `getOptions` returns only a specific field. 

Given to the name of the method, you must return all the fields, at best i guess you could pass a filter as an argument

This whole method needs to be rewritten